nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java

```diff
@@ -160,6 +160,7 @@ public Set<Element> getNonnullFieldsOfReceiverAtExit(TreePath path, Context cont
     return getNonnullReceiverFields(nullnessResult);
   }
 
+  // TODO re-use this code
```

Comment:
I added this TODO.  In this PR it'd be good if we could find a way to re-use this code, which finds all the `@NonNull` fields in a store